### DISCUSSION TITLE
Where to read code for renovatebot dashboard?

### DISCUSSION THREAD

**Discussion by mehtasankets** [renovatebot/renovate#16443] [Request Help] [answered]
Hi,

Today, I came across https://app.renovatebot.com/dashboard. For local experimentations, I would like to host this dashboard locally. But I wasn't able to find the repository for the same. Can someone let me know if the repository for above dashboard is public? If so, can someone please point me to it?

Thanks,
Sanket.

**Comment by PhilipAbed** [comment, marked as answer]
Hi. This is a service made by mend free of charge. Code is not public for the dashboard, If you want to self host you need to use your own service which u can host on github actions /docker / your own machine...etc
Read docs about self hosting.

**Comment by rarkins** [comment]
It's currently kind of hard coded to Netlify including using their oauth, so it's not designed or available to be self hosted

**Comment by mehtasankets** [comment]
Thank you for prompt responses.
